libvmaf/src/feature/float_adm.c

```diff
@@ -28,6 +28,12 @@
 #include "mem.h"
 #include "picture_copy.h"
 
+#define PROFILE_FLOAT_ADM 1 && (defined(__MINGW32__) || (!defined(_WIN32)))
```

Comment:
```suggestion
#if defined(__MINGW32__) || !defined(_WIN32)
#define PROFILE_FLOAT_ADM 1
#endif
```

Reply:
Sure, we will modify the code and push. Thanks.

Reply:
Ah I see why you had this here, although it works, it's still a bit ugly appending the conditionals at the end of a constant like this since if it was like this, you could have just done
```c++
#define PROFILE_FLOAT_ADM (defined(__MINGW32__) || !defined(_WIN32))
```
without the 1 constant while making sure this is obvious as just checking the conditionals